app/assets/stylesheets/arclight/modules/hierarchy_and_online_contents.scss

```
@@ -103,6 +103,38 @@ $hierarchy-view-collapse-icon: url("data:image/svg+xml,<svg xmlns='http://www.w3
   border-bottom: $border-width solid $border-color;
   padding-bottom: $spacer * .5;
   margin-bottom: $spacer * .75;
+  display: grid;
+  gap: 0.5rem;
+  grid-template-areas:
+    "title bookmark-toggle"
+    "request request"
+    "sidebar-toggle sidebar-toggle";
+  justify-content: start;
+  align-items: start;
+
+  @include media-breakpoint-up(lg) {
+    grid-template-areas:
+      "title request bookmark-toggle"
```

Should we stick `sidebar-toggle` in here any way? I guess I'm worried about downstream apps that might hide want to always hide that sidebar?

```suggestion
      "title request bookmark-toggle"
      "sidebar-toggle sidebar-toggle sidebar-toggle"
```

---

Sure, no harm in doing that